Neos.ContentRepository.Core/Classes/Feature/Common/NodeAggregateEventPublisher.php

```diff
@@ -38,13 +38,13 @@ public static function enrichWithCommand(
         foreach ($events as $event) {
             if ($event instanceof DecoratedEvent) {
                 $undecoratedEvent = $event->innerEvent;
-                if (!$undecoratedEvent instanceof PublishableToOtherContentStreamsInterface) {
+                if (!$undecoratedEvent instanceof PublishableToWorkspaceInterface) {
```

Comment:
I renamed the interface `PublishableToOtherContentStreamsInterface` to `PublishableToWorkspaceInterface`

Reply:
oh yes that was missed with bernhards change :D